src/rules/no_mixed_spaces_and_tabs.rs

```diff
@@ -89,6 +89,34 @@ impl LintRule for NoMixedSpacesAndTabs {
       }
     }
   }
+
+  fn docs(&self) -> &'static str {
+    r#"Disallows mixed spaces and tabs for indentation.
+
+Most code rules need to use of tabs or spaces for indentation. Therefore, if a line of code is indented with both tabs and spaces, it will usually result in an error.
```

Comment:
When reading this sentence I got a little bit confused. 

> Most code rules need to use ...

When I see "rule" I think of it as something that the syntax of a language forces developers to follow, but the language itself doesn't force it. So how about using "conventions" instead? It sounds better when referring to code styles.

> use of tabs or spaces for indentation

`of` looks unnecessary here.

> Most code rules need to use of tabs or spaces for indentation.

This whole sentence is intended to state "Most code conventions out there require developers to use _either_ tabs or spaces for indentation" right? If it's correct, let's not use "need", and instead use a verb with more strongness, like "require" or "command". Additionally, what we should clarify here is that _tabs and spaces cannot be used at the same time_, so it is much better to insert "either" into the sentence IMO.

> it will usually result in an error.

To me this phrase sounds like some runtime error will happen, but actually won't, which can be confirmed by running the following snippet where we'll get no error.

```ts
function foo() {
	 /* <---- tab and space mixed */console.log(42);
}　

foo();
```

So I'd prefer to choose `mistake` over `error` here, like `it's most likely a mistake of a developer.`